mlrun/api/api/endpoints/grafana_proxy.py

```diff
@@ -76,7 +76,8 @@ async def grafana_proxy_model_endpoints_search(
 
     :return: List of results. e.g. list of available project names.
     """
-    mlrun.api.crud.ModelEndpoints().get_access_key(auth_info)
+    if not mlrun.mlconf.is_ce_mode():
+        mlrun.api.crud.ModelEndpoints().get_access_key(auth_info)
```

Comment:
I think that in this case it is dependent on being a CE deployment because even if the user is using kafka stream (and not V3IO), he still may be using other V3IO resources.  Anyway this is some kind of an old sanity check (don't see any other reason to use it here) that should be fine in each non-CE env.